sdk/webpubsub/azure-messaging-webpubsubservice/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Release History
 
+## 1.1.0b1 (2022-11-01)
+
+### Features Added
+
+- Support sending with "filter" syntax
```

Comment:
the changelog should focus more on the changes that python sdk writers will encounter, so it should be about which parameters / functions are added in python syntax

Reply:
You can follow the pattern like: `operation xxx has new optional parameter filter`

Reply:
Updated